src/ChatBubble/index.tsx

```diff
@@ -2,41 +2,79 @@ import * as React from "react";
 import styled from "styled-components";
 import Message from "../Message";
 
-const ChatBubbleWrapper = styled.div`
+const BubbleGroupHeader = styled.h5<{ me: boolean }>`
+  margin: 0;
+  margin-bottom: 4px;
+  font-size: 14px;
+  font-weight: 500;
+  color: ${(props) =>
+    props.me
+      ? props.theme.bubbles.mineHeader
+      : props.theme.bubbles.theirHeader};
+  font-family: ${(props) => props.theme.fontFamily};
+`;
+
+const ChatBubbleWrapper = styled.div<{ me: boolean }>`
   overflow: auto;
+  max-width: 420px;
+  margin-left: ${(props) => props.me && "auto"};
 `;
 
-const ChatBubbleUI = styled.div<{ me: boolean }>`
-  background-color: ${props =>
+const ChatBubbleUI = styled.div<{ me: boolean; first?: boolean }>`
+  background-color: ${(props) =>
     props.me
       ? props.theme.bubbles.mineBackground
       : props.theme.bubbles.theirBackground};
-  border-radius: 20px;
+
+  // border-radius: 8px 8px ${(props) => (props.me ? "0 8px" : "8px 0")};
+  border-radius: ${(props) =>
```

Comment:
Is it worth moving something like this into a function outside the string template?

E.g. 
```function constructBorderRadius(props) {
  let out = "";
  if (props.me) {
  }
  ....
}
```